README.md

```diff
@@ -25,6 +25,7 @@ Options:
 * `allow_query_params`: Specifies that query string parameters will be taken into consideration when doing validation (defaults to `true`).
 * `check_content_type`: Specifies that `Content-Type` should be verified according to JSON Hyper-schema definition. (defaults to `true`).
 * `coerce_date_times`: Convert the string with `"format": "date-time"` parameter to DateTime object (default to `false`).
+* `coerce_recursive`: Convert data in array object and nested object (default to `false`).
```

Comment:
Just OOC, is the extra option for backwards compatibility or something? It seems to me that having coercion that'll recurse into your more complex objects is probably the default that most users would want.

Reply:
Yes, this option for backwards compatibility...
However, 2.0.0 is major version up, so I think it is okay to default true.
What do you think?

(In other words, we want to consider the default true for other options as well)

Reply:
Good thinking on the backwards compatibility!

Yes, lets ship this as part of 2.0.0 though. I think it's best as a default, and we're in the process of a major version bump anyway. I'll make sure to note it conspicuously in the CHANGELOG. Can you make the change? Thanks!